libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -91,61 +91,49 @@ class AC_AttitudeControl {
     virtual const AC_PID& get_rate_yaw_pid() const = 0;
 
     // get the roll acceleration limit in centidegrees/s/s or radians/s/s
-    float get_accel_roll_max_cdss() const { return _accel_roll_max_cdss; }
     float get_accel_roll_max_radss() const { return cd_to_rad(_accel_roll_max_cdss); }
 
     // Sets the roll acceleration limit in centidegrees/s/s
```

Comment:
Lots of the comments in this file need updating to match the changed methods..
```suggestion
    // Sets the roll acceleration limit in radians/s/s
```